src/main/java/edu/whimc/portals/Portal.java

```diff
@@ -23,37 +23,90 @@
 import org.bukkit.permissions.Permission;
 import org.bukkit.util.Vector;
 
+/**
+ * The class to sore Portal data.
+ */
 public class Portal {
 
+    /* The list of portals. */
```

Comment:
```suggestion
    /* The list of all portals. */
```
Same comment for the getPortals doc